File: Find a Pair/solve with js/solve-1.js

// Time complexity: O(n²)
// Space complexity: O(1)




function findPair(arr,k){
	for (let i = 0; i < arr.length; i++) {
		for (let j = 0; i+1 < arr.length; j++) {
			if(arr[i] + arr[j] == k) return [i,j];
		}
			
		}
		return false;
	}

	

File: Find a Pair/solve with c++/solve-1.cpp
bool findPair(int* arr, int arrLength, int k){
  for(int i = 0; i < arrLength; i++){
    for(int j = i+1; j < arrLength; j++){
      if(arr[i] + arr[j] == k) return true;
    }
  }
  return false;
}

